Corona-Warn-App/src/main/res/layout/event_registration_category_fragment.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/category_root"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:contentDescription="@string/tracelocation_organizer_category_title">
+
+        <include
+            android:id="@+id/category_navbar"
+            layout="@layout/include_header"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:icon="@{@drawable/ic_close}"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:title="@{@string/tracelocation_organizer_category_title}" />
```

Comment:
can you use the `MaterialToolbar` tag directly I don't see any value of using this <include> tag, we have also styles for the toolbar

Reply:
Oh nice, using `MaterialToolbar` now.